src/components/DepositWidget/Row.tsx

```diff
@@ -35,6 +42,20 @@ function _loadFallbackTokenImage(event: React.FormEvent<HTMLImageElement>): void
   image.src = unknownTokenImg
 }
 
+const txOptionalParams: TxOptionalParams = {
+  onSentTransaction: (receipt: Receipt): void => {
+    toast.info(
+      <div>
+        The transaction has been sent! Check{' '}
+        <a href={`https://etherscan.io/tx/${receipt.transactionHash}`} target="_blank" rel="noopener noreferrer">
+          {receipt.transactionHash.slice(0, 6)}...{receipt.transactionHash.slice(-4)}
```

Comment:
This slide it's better done in a util. 
It can be sth like:
- You provide a string and a max size
- If the string is short enough, returns the original
- Otherwise, adds trims it and add the ellipsis

Reply:
Done.